### PR TITLE
Add fetch logic that fetches hardcoded data

### PR DESCRIPTION
**Pull request by limli**
Add ajax call to load hardcoded data from /data

### CODE REVIEW

**Comment by SuyashLakhotia** [comment]
Might be useful to reword the title of your PR to reflect the changes instead of the STEP walkthrough section.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello limli");
```
Comment (SuyashLakhotia) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Is the removal of `<h1>` tags intentional here?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
...
   greetingContainer.innerText = greeting;
 }
+
+function loadData() {
+    const commentsContainer = document.getElementById('comments-container');
+    commentsContainer.innerText = "Loading comments..."
+    fetch("/data").then(response => response.text()).then(mytext => {
+        commentsContainer.innerText = mytext;
+    });
+}
```
Comment (SuyashLakhotia) [inline portfolio/src/main/webapp/script.js:36]: nit: add a newline at the end of the file

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadData() {
+    const commentsContainer = document.getElementById('comments-container');
```
Comment (SuyashLakhotia) [inline portfolio/src/main/webapp/script.js:31]: nit: indentation should be two spaces, not four

https://google.github.io/styleguide/jsguide.html#formatting-block-indentation

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (limli) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: yes

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (limli) [reply portfolio/src/main/webapp/script.js]: I've installed eslint to fix the style automatically

**Comment by SuyashLakhotia** [comment]
Please merge #4 into `master` before this PR.

**Pull request merged**